### PR TITLE
Added new module for denmark locale

### PR DESCRIPTION
**Pull request by odoohouse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
Hello,

I leave the accounting team review the content itself but it seems you have not [signed the CLA](https://www.odoo.com/sign-cla).
Could you add it so we can integrate your patch?
Regards

**Comment by odoohouse** [comment]
We have added Odoo CLA signature file and raised pull request: https://github.com/odoo/odoo/pull/23731

**Comment by hdinsen** [comment]
@mart-e 
We have signed the CLA

**Comment by odoohouse** [comment]
done rebase and updated pull request

File: addons/l10n_dk/data/account_deafults.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (smetl) [inline addons/l10n_dk/data/account_deafults.xml]: rename `account_deafults.xml` to `account_defaults.xml`

File: addons/l10n_dk/data/account_chart.xml [hunk 2]
```diff
@@ -0,0 +1,1552 @@
...
+    </record>
+    <record id="a7705" model="account.account.template" >
+      <field name="code">7705</field>
+      <field name="reconcile" eval="False" />
+      <field name="user_type_id" ref="l10n_dk.account_type_credit_institutions" />
+      <field name="name">Realkreditinstitutter</field>
+      <field name="chart_template_id" ref="dk_chart_template"/>
+    </record>
+    <record id="a7710" model="account.account.template" >
+      <field name="code">7710</f
```
Comment (smetl) [inline addons/l10n_dk/data/account_chart.xml]: fix indent

Comment (smetl) [inline addons/l10n_dk/data/account_chart.xml]: add new line in all files

File: addons/l10n_dk/data/account_chart_template.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+-
+  !python {model: account.chart.template, id: dk_chart_template}: |
+    self[0].try_loading_for_current_company()
```
Comment (smetl) [inline addons/l10n_dk/data/account_chart_template.yml]: `.yml` is not deprecated. Please do it on an `.xml` file containing:

```
<?xml version="1.0" encoding="utf-8"?>
<odoo>
    <data noupdate="1">
        <function model="account.chart.template" name="try_loading_for_current_company">
            <value eval="[ref('l10n_dk.dk_chart_template')]"/>
        </function>
    </data>
</odoo>
```

File: addons/l10n_dk/data/account_deafults.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+<odoo>
+
```
Comment (smetl) [inline addons/l10n_dk/data/account_deafults.xml]: Could you please add the `<data></data>` layer plz?

File: addons/l10n_dk/data/account_financial_report.xml [hunk 5]
```diff
@@ -0,0 +1,619 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
```
Comment (smetl) [inline addons/l10n_dk/data/account_financial_report.xml]: ´noupdate="0"´ by default so you can remove this attribute

File: addons/l10n_dk/data/account_financial_report.xml [hunk 6]
```diff
@@ -0,0 +1,619 @@
...
+            Financial Reports
+            display_detail:
+            no_detail
+            detail_flat
+            detail_with_hierarchy
+        -->
+        
+    <!--Delete all old entries-->
+    <delete model="account.financial.report" search="[('type','=','account_report')]"/>
+    <delete model="account.financial.report" search="[('id','>',0)]"/>
```
Comment (smetl) [inline addons/l10n_dk/data/account_financial_report.xml]: Existing reports MUST not be deleted. These lines must be removed.

File: addons/l10n_dk/data/account_financial_report.xml [hunk 7]
```diff
@@ -0,0 +1,619 @@
...
+
+    <record id="account_financial_report_resultat" model="account.financial.report">
```
Comment (smetl) [inline addons/l10n_dk/data/account_financial_report.xml]: All the financial reports must be added as dynamic reports in a dedicated enterprise module called ´l10n_dk_reports´.

File: addons/l10n_dk/data/account_fiscal_position_account_template.xml [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+        </record>
+        <record
+            id="fiscal_position_account_koebydelser_eu_taxid"
+            model="account.fiscal.position.account.template">
+            <field name="position_id"     ref="fiscal_position_template_eu_taxid" />
+            <field name="account_src_id"  ref="a2011" />
+            <field name="account_dest_id" ref="a2021" />
+        </record>
+    </data>
+</odoo>
```
Comment (smetl) [inline addons/l10n_dk/data/account_fiscal_position_account_template.xml]: add new line

File: addons/l10n_dk/data/account_tax.xml [hunk 9]
```diff
@@ -0,0 +1,370 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- (c) 2018 Odoo House ApS -->
```
Comment (smetl) [inline addons/l10n_dk/data/account_tax.xml]: add this comment on top and fix the file indent

File: addons/l10n_dk/data/account_account_types.xml [hunk 10]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
+     (c) 2018 Odoo House ApS
+     <field name="type">receivable</field>
+     <field name="type">payable</field>
+     <field name="type">liquidity</field>
+     <field name="type">other</field>
+    -->
+    <data noupdate="True">
```
Comment (smetl) [inline addons/l10n_dk/data/account_account_types.xml]: Are you sure all these new types are necessary? Some of them seems to be duplicates of types already defined in https://github.com/odoo/odoo/blob/master/addons/account/data/data_account_type.xml

For example:

l10n_dk.account_type_revenue == account.data_account_type_revenue?
l10n_dk.account_type_prepayments == account.data_account_type_prepayments?
l10n_dk.account_type_direct_costs == account.data_account_type_direct_costs?
l10n_dk.account_type_depreciation == account.data_account_type_depreciation?

**Comment by qdp-odoo** [comment]
@odoohouse allo? is there someone in here?

**Comment by smetl** [comment]
As the PR's author is not responding, I close the PR until we find a guy ready to implement the DE localization.

**Comment by odoohouse** [comment]
Tonight we commit last changes

**Pull request closed without merging**